octoprint_PrintTimeGenius/static/js/PrintTimeGenius.js

```diff
@@ -70,19 +70,25 @@ $(function() {
         }
       }
       return results;
-    }
+    };
 
-    self.FileList = ko.pureComputed(function() {
-      return self.recurseFiles(self.filesViewModel.allItems()).slice()
-          .sort(function(a,b) {
-            if (_.has(a, "gcodeAnalysis.progress") !=
-                _.has(b, "gcodeAnalysis.progress")) {
-              return (_.has(a, "gcodeAnalysis.progress") -
-                      _.has(b, "gcodeAnalysis.progress"));
-            }
-            return a.path.localeCompare(b.path);
-          });
-    });
+    self.theFiles = function(items) {
```

Comment:
Since we no longer call recurseFiles, we can delete it, right?

Reply:
yep!